test/integration/credentials-auth.spec.js

```diff
@@ -87,7 +87,6 @@ describe(`integration with ${authType}`, async function () {
     tempDb = getDb(tempDbName, auth, authType);
 
     const collectLogs = await utils.getDockerContainerLogs();
-    await db.allDocs();
```

Comment:
Sometimes this request ended up polluting the docker logs with an extra session request, because in CI the cookie that we generated in a previous test would have expired already.